meos/include/meos.h

```diff
@@ -1328,7 +1332,7 @@ extern Temporal *temporal_scale_time(const Temporal *temp, const Interval *durat
 extern Temporal *temporal_set_interp(const Temporal *temp, interpType interp);
 extern Temporal *temporal_shift_scale_time(const Temporal *temp, const Interval *shift, const Interval *duration);
 extern Temporal *temporal_shift_time(const Temporal *temp, const Interval *shift);
-extern Temporal *temporal_to_tinstant(const Temporal *temp);
+extern TInstant *temporal_to_tinstant(const Temporal *temp);
 extern Temporal *temporal_to_tsequence(const Temporal *temp, interpType interp);
 extern Temporal *temporal_to_tsequenceset(const Temporal *temp, interpType interp);
```

Comment:
Why does `temporal_to_tinstant` return a `TInstant`, while `temporal_to_tsequence` and `temporal_to_tsequenceset` still return a `Temporal`